### DISCUSSION TITLE
Multiple Checkpoints

### DISCUSSION THREAD

**Discussion by thazir** [deepmodeling/deepmd-kit#356] [Q&A] [answered]
I read in the manual that "checkpoints will be written to files with prefix save_ckpt every save_freq batches". Still I see that only one checkpoint file is saved. Am I doing something wrong?

**Comment by njzjz** [comment, marked as answer]
It will override the previous checkpoint...

**Reply by thazir** [reply]
Thanks a lot for the prompt reply. Still i think it would be nice if multiple checkpoints were saved in case some previous training step was better than the last saved.
